tests/suites/test_suite_nist_kw.data

```diff
@@ -45,6 +45,15 @@ nist_kw_ciphertext_lengths:16:MBEDTLS_KW_MODE_KW:MBEDTLS_ERR_CIPHER_BAD_INPUT_DA
 NIST KW lengths #9 KWP ciphertext too short (2 to 2^29 semiblocks)
 nist_kw_ciphertext_lengths:8:MBEDTLS_KW_MODE_KWP:MBEDTLS_ERR_CIPHER_BAD_INPUT_DATA
 
+NIST KW lengths #10 KW plaintext invalid length (2 to 2^54 - 1 semiblocks)
```

Comment:
The test descriptions for `nist_kw_ciphertext_lengths` and `nist_kw_plaintext_lengths` don't correspond to what the tests do. For example this test uses an input length of 21, but this is not reflected in the test description. The test description should say “21 octets” or “21 bytes” or “input length is not a whole number of semiblocks” or something.